src/Microsoft.DotNet.Interactive.Tests/LanguageKernelPackageTests.cs

```diff
@@ -235,6 +235,50 @@ public async Task Pound_r_nuget_disallows_empty_package_specification(Language l
             .Be("(1,4): error DNI210: Unable to parse package reference: \"nuget:\"");
     }
 
+    [Fact]
+    public async Task Pound_r_is_not_treated_as_pound_r_nuget_by_csharp_kernel_if_assembly_path_happens_to_contain_the_string_nuget()
+    {
+        var kernel = CreateCSharpKernel();
+
+        var result =
+            await kernel.SubmitCodeAsync(
+                """
+                #r "C:\Users\abcde\.nuget\packages\package\1.0.0\package.dll"
+                """);
+
+        result.Events
+            .Should()
+            .ContainSingle<CommandFailed>()
+            .Which
+            .Message
+            .Should()
+            .ContainAll("Metadata file", "could not be found")
+            .And
+            .NotContain("DNI");
+    }
+
+    [Fact]
+    public async Task Pound_r_is_not_treated_as_pound_r_nuget_by_fsharp_kernel_if_assembly_path_happens_to_contain_the_string_nuget()
+    {
+        var kernel = CreateKernel(Language.FSharp);
+
+        var result =
+            await kernel.SubmitCodeAsync(
+                """
+                #r "C:\Users\abcde\.nuget\packages\package\1.0.0\package.dll"
```

Comment:
```suggestion
                #r @"C:\Users\abcde\.nuget\packages\package\1.0.0\package.dll"
```

Reply:
Thanks for catching - fixed (and also changed the expected error)